docs/modules/ROOT/pages/explanations/commodore-components.adoc

```diff
@@ -0,0 +1,49 @@
+= Commodore component best practices
+
+== Handling major changes
```

Comment:
Another suggestion for a new section: Handling secrets like we do now with https://github.com/appuio/component-openshift4-authentication/pull/32

Reply:
added in latest version